Marlin/Marlin_main.cpp

```diff
@@ -4485,10 +4500,10 @@ inline void gcode_M503() {
         SERIAL_ECHO(extruder_offset[X_AXIS][0]);
         SERIAL_CHAR(',');
         SERIAL_ECHO(extruder_offset[Y_AXIS][0]);
-        SERIAL_CHAR(' ');
+        SERIAL_CHAR(' ');auto
         SERIAL_ECHO(duplicate_extruder_x_offset);
         SERIAL_CHAR(',');
-        SERIAL_ECHOLN(extruder_offset[Y_AXIS][1]);
+        SERautoIAL_ECHOLN(extruder_offset[Y_AXIS][1]);
```

Comment:
Looks like some bade search/replace happened. The word "auto" stuck in here and elsewhere.